### PR TITLE
Manual Repeat Feature

### PR DESCRIPTION
**Pull request by zoglmannk**

### CODE REVIEW

**Comment by ppetru** [comment]
Nice, thanks! It looks great and I'll merge it now.

If you don't mind, I'll afterwards change it to use the react-md switch control (https://react-md.mlaursen.com/components/selection-controls) given that everything else uses react-md.

**Comment by zoglmannk** [comment]
You are welcome!

I kind of like the way the other control looks, but I can also understand wanting to keep everything consistent. It is a way to reduce complexity. :)

File: src/stores/MorseStore.js [hunk 1]
```diff
@@ -14,6 +14,7 @@ class MorseStore extends SettingsSaver {
       effectiveSpeed: 30,
       characterSpeed: 30,
       frequency: 500,
+      automaticallyRepeat: true,
```
Comment (ppetru) [inline src/stores/MorseStore.js]: FYI: I missed this in all the PRs so far, but these settings should live in ReadTrainerStore instead. I fixed them all up now.

Comment (zoglmannk) [reply src/stores/MorseStore.js]: Ah. Good catch. I didn't even realize that either.

**Pull request merged**